src/content/docs/github/github-actions/github-actions-cicd.md

```diff
@@ -0,0 +1,290 @@
+
+| Feature | Description |
+| --- | --- |
+| Automation of Workflows: | GitHub Actions allows to automate software development workflows directly in GitHub repository. User can set up workflows to build, test, package, release, and deploy code based on a variety of triggers, such as pushing to a repository or making a pull request. |
+| Customizable Workflows: | User can create custom workflows using the YAML syntax to define the steps and tasks that run on specific triggers. This provides flexibility to tailor automation processes to meet specific project needs. |
+| Hosted Runners: | GitHub provides hosted virtual environments (runners) where these workflows execute. These runners can be based on Ubuntu Linux, Windows, and macOS. Additionally, user can host own runners for more control and customization. |
+| Marketplace Integration: | GitHub Actions integrates with the GitHub Marketplace, allowing users to find and share actions to perform tasks, ranging from code linting and testing to deploying to various cloud providers. |
+| CI/CD Support: | GitHub Actions supports Continuous Integration and Continuous Deployment (CI/CD) practices, enabling automated testing and deployment with each code change, helping to maintain code quality and accelerate the deployment cycle. |
+| Community and Sharing: | Actions created by the community can be shared and reused across different projects. This fosters a collaborative environment where developers can leverage each other's expertise to streamline development processes. |
+| Secrets and Encrypted Keys: | It supports secrets management, allowing user to store sensitive information like API keys or access tokens securely in the GitHub environment. |
+| Example Feature | This feature does something for a specific reason. |
+
+These features make GitHub Actions a versatile tool for developers looking to automate their development and deployment workflows directly within GitHub. For more detailed information and guidance on setting up GitHub Actions, you can visit the [documentation][2]. 
+
+
+## Market Comparison
+
+I choose the following two platforms: GitHub Actions && Azure Pipelines because they are both owned by Microsoft. <br> 
+
+Azure DevOps and GitHub have their own automation and CI/CD (continuous integration and continuous delivery) tools: Azure Pipelines and GitHub Actions.
+Both of these tools allow you to create workflows for building, testing, publishing, releasing, and deploying code. They also have similarities in terms of workflow setup.
+In both cases, configuration files are saved in YAML format and stored in the code repository. Workflows consist of one or more tasks, and each of these tasks contains one or more individual steps or commands.
+
+Task and step structures in Azure Pipelines are very similar to those in GitHub actions.
```

Comment:
Spacing issue here ```suggestion
Task and step structures in Azure Pipelines are very similar to those in GitHub actions.

```

Reply:
Hi,
@sulenchy I will have access to computer on Monday and will correct all of fails